tests/test-that-all-files-are-executable.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+require 'rspec'
+
+RSpec.configure do |config|
+  config.formatter = :documentation
+end
+
+context "All Commands Should Be Executable" do
```

Comment:
Might be worth adding a `# comment` to explain why this test was not done as part of `environment.bats` (and maybe one in `tests/environment.bats` as well to point to this file) and why this had to be moved to Ruby.

Otherwise I can totally see one of our future selves being tempted to move it back to `environment.bats` in order to mutualize all the Unit Tests in the same place, without remembering about the quirk we discovered about it.

Reply:
Yep good call – added in 27d6c20f119c53574960b43353b59c162b0eb286